tests/unit/ResolutionDates.spec.ts

```diff
@@ -307,7 +307,7 @@ describe('Resolution Dates component - edit mode', () => {
   it('displays the Correct button for correction filings', () => {
     store.stateModel.tombstone.entityType = CorpTypeCd.BENEFIT_COMPANY
     store.stateModel.tombstone.filingType = FilingTypes.CORRECTION
-    store.stateModel.shareStructureStep.shareClasses = shareClasses
+    store.stateModel.shareStructureStep.shareClasses = shareClasses as any
```

Comment:
If I didn't have these in, would complain about missing properties

Reply:
Beauty.